static/app/components/events/interfaces/performance/resources.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled';
+
+import {IconDocs} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+import EventDataSection from '../../eventDataSection';
+
+export type ResourceLink = {
+  link: string;
+  text: string;
+};
+
+type Props = {
+  description: string;
+  links: ResourceLink[];
+};
+
+// This section provides users with resources on how to resolve Performance Issues
```

Comment:
maybe make this comment less specific to performance issues? especially if you want this to be a generic component

Reply:
yep true, good point